DEVELOPMENT.md

```diff
@@ -35,45 +35,54 @@ Install these tools:
     ```bash
     kind create cluster
     ```
+   **NOTE**: The KinD cluster can also be created with tools like 
+   [Podman Desktop](https://podman-desktop.io/docs/kind/creating-a-kind-cluster).
+ 
 
 1. Set up `ko` to upload images to the KinD cluster, or any other registry you
   want to use:
     ```bash
     export KO_DOCKER_REPO="kind.local"
     ```
+   **NOTE**: if your cluster doesn't have the name `kind` you need to specify the env var `KIND_CLUSTER_NAME`.
 
 ## Install KubeArchive
 
 1. Use Helm to install KubeArchive:
-    ```bash
-    helm install kubearchive charts/kubearchive \
-        --set-string apiServer.image=$(ko build github.com/kubearchive/kubearchive/cmd/api) \
-        --set-string sink.image=$(ko build github.com/kubearchive/kubearchive/cmd/sink)
-    ```
+   ```bash
+   helm install kubearchive charts/kubearchive -n default \
```

Comment:
This is not needed. `-n` populates the `{{ .Release.Namespace }}` variable which we are not using. We also don't want to install anything on default. If anything, this should be `-n kubearchive`, but again we are not using the variable.

https://helm.sh/docs/chart_template_guide/builtin_objects/

Reply:
I'm confused about this. AFAIU, when running `helm install` the chart will be installed in a namespace of the kubernetes cluster. If we run this just after we deploying the KinD cluster, the kubernetes context will be the `default` namespace so the chart will be installed there.

I think that it is better to explicitly state the namespace.

Note that after installing a chart, it can be listed by running `helm  list`. If you aren't in the same namespace that you where when running `helm install`, you won't be able to see the chart. 

If we add the namespace in all the instructions, they become more "reproducible" and prevent finding errors.

Reply:
After testing a little bit... even if we declare `-n` to be default, the chart is installed on the kubearchive namespace as we have it coded in the files. If we want to use `-n` to install the chart somewhere we need to change our implementation.

So I think that both having `-n default` and not having it are not correct, because our implementation does not make good use of it. I suggest we don't specify it and address the problem in a separate issue.

Reply:
After talking with Marta, let's keep this with `-n default` as Marta introduced. For me its confusing that `-n` does not impact our resources, but not specifying it can lead to confusion when updating the helm chart if the user moved from the original namespace.